Makefile

```diff
@@ -11,9 +11,6 @@ integration-test:
 pygoat-test:
 	${PYTEST} -v ci_tests/test_pygoat_findings.py
 
-lint:
-	pylint -v codemodder core_codemods tests integration_tests
-
```

Comment:
Any reason for not replacing this with the `ruff` equivalent?

Reply:
Well... I figured since it runs in your pre-commit there isn't much of a point? Do you strongly want it?

Reply:
I use this command a lot, it would be good to keep around if possible.